tests/tiny.nf.test

```diff
@@ -0,0 +1,26 @@
+nextflow_pipeline {
+
+    name "Test Workflow main.nf"
+    script "../main.nf"
+    config "../conf/tests/test_tiny.config"
+
+    test("Tiny dataset should run without failures and produce expected files") {
```

Comment:
this line should conclude what the parameters are (when we have more tests it is easier to track. )